forms-flow-web-root-config/public/config/config.template.js

```diff
@@ -63,5 +63,6 @@ window["_env_"] = {
   REACT_APP_ENABLE_APPLICATIONS_MODULE: "${REACT_APP_ENABLE_APPLICATIONS_MODULE}",
   REACT_APP_DATE_FORMAT:"${REACT_APP_DATE_FORMAT}",
   REACT_APP_TIME_FORMAT:"${REACT_APP_TIME_FORMAT}",
-  REACT_APP_USER_NAME_DISPLAY_CLAIM: "${REACT_APP_USER_NAME_DISPLAY_CLAIM}"
+  REACT_APP_USER_NAME_DISPLAY_CLAIM: "${REACT_APP_USER_NAME_DISPLAY_CLAIM}",
+  REACT_APP_GRAPHQL_API_URL: "${GRAPHQL_API_URL}" 
```

Comment:
here we need to add prefix as REACT_APP like this
REACT_APP_GRAPHQL_API_URL: "${REACT_APP_GRAPHQL_API_URL}"

Reply:
config.sample.js